sshoot/config.py

```diff
@@ -21,7 +21,7 @@ def yaml_dump(data: Dict, fh: Optional[IO] = None):
 class Config:
     """Handle configuration file loading/saving."""
 
-    CONFIG_KEYS = frozenset(["executable"])
+    CONFIG_KEYS = frozenset(["executable", "global_extra_opts"])
```

Comment:
can you please rename the config key to `extra-options`?

Reply:
Done :slightly_smiling_face: